eventsapi.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"time"
+)
+
+func main() {
+	var api_token = "Bearer " + os.Getenv("EVENTS_API_TOKEN")
+	var url = "https://events.1password.com"
+
+	var start_time = time.Now().AddDate(0, 0, -24)
```

Comment:
This will subtract 24 days, not hours.
```suggestion
	var start_time = time.Now().Add(-24 * time.Hour)
```

Reply:
Never mind, I see @anthonythleung caught this in his wider suggestion below!